migrations/2023-10-19-102636_back_fill_n_remove_connector_response/up.sql

```diff
@@ -0,0 +1,11 @@
+-- Your SQL goes here
+ALTER TABLE payment_attempt 
+ADD COLUMN authentication_data JSON, 
+ADD COLUMN encoded_data TEXT;
+
+UPDATE payment_attempt 
+SET (authentication_data, encoded_data) = (connector_response.authentication_data, connector_response.encoded_data) 
+from connector_response 
+where payment_attempt.payment_id = connector_response.payment_id 
+    and payment_attempt.attempt_id = connector_response.attempt_id
+    and payment_attempt.merchant_id = connector_response.merchant_id;
```

Comment:
Do we need to backfill this on payments which already succeeded or failed?

Reply:
It is better to do, Since we don't delete connector response data for succeeded or failed payments.

Reply:
Yes this will have downtime, instead do this in batches based on the date

Reply:
It is already discussed here  . https://github.com/juspay/hyperswitch/pull/2656#discussion_r1368468789